src/index.html

```diff
@@ -6,7 +6,44 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
-  </body>
+  <div class="calendar">
+    <div class="calendar__start-day-wed"></div>
+    <div class="calendar__month-length-29"></div>
+
+    <div class="calendar__sheet" >
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+      <div class="calendar__sheet-cell"></div>
+    </div>
+  </div>
+
+
 </html>
```

Comment:
```suggestion
  </div>
</html>
```
Don't leave empty lines between parent and child elements. Also, body is absent